assets/src/edit-story/utils/test/cleanForSlug.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import cleanForSlug from '../cleanForSlug';
+
+describe( 'cleanForSlug', () => {
+	it( 'should return string prepared for use as URL slug', () => {
+		expect( cleanForSlug( ' /Déjà_vu. ' ) ).toBe( 'deja-vu' );
```

Comment:
Same tests as for the original function:

https://github.com/WordPress/gutenberg/blob/06bd2662c8fafacfbd6832aa6017b91915375445/packages/editor/src/utils/test/url.js#L6-L18

Reply:
Can we put a comment in the code to reflect where these tests came from.

Reply:
It's only this one test really. I think the new docblock on the actual function covers this.